source/crc32.c

```diff
@@ -0,0 +1,72 @@
+#include <aws/cal/crc32.h>
+#include <aws/cal/hash.h>
+#include <aws/checksums/crc.h>
+
+/* hash is object, vtable is methods, hash properties (not vtable) are data?*/
+
+static void s_destroy(struct aws_hash *hash);
+static int s_update(struct aws_hash *hash, const struct aws_byte_cursor *to_hash);
+static int s_finalize(struct aws_hash *hash, struct aws_byte_buf *output);
+/* why do we need to define and implement the functions separatly? */
+/* can we reuse some of these as defaults defined in hash.c?*/
```

Comment:
no because they're declared static, which keeps them only visible to the translation unit they're in (e.g. their C file)

Reply:
Can remove this comment